org.opent2t.sample.hub.superpopular/com.wink.hub/js/thingTranslator.js

```diff
@@ -203,18 +210,20 @@ class Translator {
                         // We already have this data, so no need to make an unnecesary request over the wire.
                         return OpenT2T.invokeMethodAsync(translator, opent2tInfo.schema, 'get', [expand, winkDevice])
                             .then((platformResponse) => {
-                                return platformResponse; 
+                                return platformResponse;
                             });
                     }));
             }
         });
 
         // Return a promise for all platform translations.
-        return Promise.all(platformPromises)
-            .then((platforms) => {
-                var toReturn = {};
-                toReturn.schema = "org.opent2t.sample.hub.superpopular";
-                toReturn.platforms = platforms;
+        return Promise.all(platformPromises.map(promiseReflect))
```

Comment:
i see -- cool!  Still good to have a test case for it 

---
In reply to: [107275217](https://github.com/openT2T/translators/pull/216#discussion_r107275217) [](ancestors = 107275217,107274687)